.github/workflows/auto-sync+bump-date.yml

```diff
@@ -0,0 +1,56 @@
+name: Sync and Merge with MarlinFirmware/Marlin
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron:  "* */24 * * *"
+  
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Sync and merge upstream repo
+  # You may pin to the exact commit or the version.
+  # uses: dabreadman/sync-upstream-repo@fc5fe9952946b1daaafd9abd7fcd7e260b81ddbe
+      uses: dabreadman/sync-upstream-repo@v1.3.0
+      with:
+    # URL of gitHub public upstream repo
+        upstream_repo: https://github.com/MarlinFirmware/Marlin.git
+    # Branch to merge from upstream (defaults to downstream branch)
+        upstream_branch: bugfix-2.1.x
+    # Branch to merge into downstream
+        downstream_branch: bugfix-2.1.x
+    # GitHub Bot token
+        token: ${{ secrets.GITHUB_TOKEN }}
+        
+  notify:
+    needs: sync
+    name: Bump Distribution Date
+    if: github.repository == 'classicrocker883/Marlin'
```

Comment:
Based on this line, this workflow will only run in your fork. I'm not sure if this workflow would be appropriate to include here/upstream.